### DISCUSSION TITLE
Build with configuration file for package

### DISCUSSION THREAD

**Discussion by beu-adesso** [python-poetry/poetry#8087] [Q&A] [answered]
Hello,
I'm developing a programm, that needs a configuration file.
I think it would be good if poetry would install this in `/etc/` on linux, so that the programm can access this and the user can edit this after installation of the package.
Is this possible with poetry or is there a better solution for this problem?

**Comment by Secrus** [comment, marked as answer]
This is not something that any standard Python package installer will do. The best way to do it would be for you to provide users with instructions on how to create a config file, or (if your package provides CLI) add a command to generate a template config file.

**Reply by beu-adesso** [reply]
Ok, but if I'm understanding it right it is possible with [setuptools](https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#id51), so is there a way for poetry to do the same?

**Reply by dimbleby** [reply]
you are not understanding it right, even `data_files` does not support absolute paths.

If you want to put files in `/etc/` you should build an rpm or deb or whatever package is appropriate for the systems you are targetting.  

Probably you are better off just not doing that, though - have your program behave sensibly when no configuration file is present, and tell users how and where to create one if they need to.
